### DISCUSSION TITLE
CaIman plot values

### DISCUSSION THREAD

**Discussion by LeoCAdv** [flatironinstitute/CaImAn#1400] [Q&A]
Hello, 
I'm wondering if someone may discuss with me how I can create individual strings containing the plot values for each neuron? I wish to re-make all the waveforms on excel/graphpad and compute an average waveform.

**Comment by kushalkolar** [comment]
Strings? I don't understand the question sorry.

**Comment by LeoCAdv** [comment]
I was  able to resolve this.
